manager/src/test/java/com/redhat/service/smartevents/manager/ProcessorServiceConnectorTest.java

```diff
@@ -295,7 +285,7 @@ private void assertShardAsksForProcessorToBeDeletedIncludes(Processor processor)
 
     private void assertShardAsksForProcessorToBeDeletedDoesNotInclude(Processor processor) {
         List<Processor> processorsToBeDeleted = processorDAO.findByShardIdWithReadyDependencies(TestConstants.SHARD_ID);
-        assertThat(processorsToBeDeleted.stream().map(Processor::getId)).doesNotContain(processor.getId());
+        assertThat(processorsToBeDeleted.stream().map(Processor::getId)).isNotEmpty().doesNotContain(processor.getId());
```

Comment:
Seems that some tests are failing on this change

Reply:
> Seems that some tests are failing on this change
OK, will fix it.